dotnet3.5/S3/UploadObjectExample/UploadObject/UploadObject.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier:  Apache - 2.0
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace UploadObject
+{
+    // The following example uploads an object to an Amazon Simple Storage
+    // Service (Amazon S3) bucket in two different ways. First, it uploads
+    // an existing file from the supposed file path and then it creates an
+    // object in the example bucket by putting text into the named file. It
+    // was created using AWS SDK for .NET 3.5 and .NET 5.0.
+
+    class UploadObject
+    {
+        // Specify your AWS Region (an example Region is shown).
```

Comment:
This comment provides no value.

Reply:
Yes, it does. The point of having the comment there is to remind people to change the region to their own. Hopefully, the fact that the ccmment is there will be enough of a reminder.